auditor/src/domain/component.rs

```diff
@@ -15,15 +15,42 @@ use sqlx::{
     Postgres, Type,
 };
 
+/// A [`Component`] represents a single component that is to be accounted for.
+///
+/// It consists at least of a `name` and an `amount` (how many or how much of this component is to
+/// be accounted for).
```

Comment:
Nitpick: "A component has an associated `name` and `amount` (...."

Reply:
Fixed